### ISSUE TITLE
Discussion - Adding Hugging Face access token and username as input arguments 

### ISSUE THREAD

**Issue by anandhu-eng** [mlcommons/cm4mlops#400]
Would it be helpfull if we add Hugging Face access token and username as input arguments as it decreases the level of user interaction required when running the benchmarks for private models. The login could be handled in [get-ml-model-huggingface-zoo](https://github.com/mlcommons/cm4mlops/blob/main/script/get-ml-model-huggingface-zoo/_cm.json).

**Comment by arjunsuresh** [comment]
yes @anandhu-eng . We can actually have `authenticate-huggingface-cli` script similar to `authenticate-github-cli` and add it as a dependency to `get-ml-model-huggingface-zoo`.
